gitbutler-ui/src/lib/components/BranchCard.svelte

```diff
@@ -99,16 +101,12 @@
 			.join('\n')
 			.slice(0, 5000);
 
-		if (user && aiGenEnabled) {
-			const aiProvider = new ButlerAIProvider(cloud, user);
-
-			new Summarizer(aiProvider).branch(diff).then((message) => {
-				if (message !== branch.name) {
-					branch.name = message;
-					branchController.updateBranchName(branch.id, branch.name);
-				}
-			});
-		}
+		summarizer.branch(diff).then((message) => {
```

Comment:
Are you intentionally using `then` or would this be better written with `await`?

Reply:
When I modified that part of the function, it wasn't yet declared as `async` but now it is so I'll move it over; thanks for the spot 👍